project.clj

```diff
@@ -18,6 +18,10 @@
                  [byte-streams "0.2.1"]
                  [camel-snake-kebab "0.3.2"]
 
+                 [clj-http "2.1.0"]
+                 [clj-http-fake "1.0.2"]
+                 [clj-time "0.11.0"]
+
```

Comment:
Didn't know about aleph. (I could look at my Google history but I think this was at the top of "Clojure HTTP request library" or something.)

Should I switch it over? (And does aleph have a request mocking library that goes along with it, because I think clj-http-fake is just for clj-http.)